Update project readme

MuriloKakazu opened the pull request:
Update project readme to show this project's description and deploy instructions

MuriloKakazu commented:
@mupapi Please review this **pr** after accepting & merging our new **circle-ci implementation**

mupapi commented:
@MuriloKakazu Oh sh**, I believe the CI isn't deleting the scratch orgs when tests are over. It just reached the actives scratch orgs limit

MuriloKakazu commented:
> @MuriloKakazu Oh sh**, I believe the CI isn't deleting the scratch orgs when tests are over. It just reached the actives scratch orgs limit

We have a limit of 3 active scratch orgs (Trial Edition)... 2 of them were mine (I've deleted one of them now since I'm not using it)..... Since you merged the last pr and updated this one, CI enqueued 2 jobs, which would create 2 more scratch orgs, summing up to 4, which passes out limit! 😢
